tutormyplugin/patches/openedx-lms-development-settings

```diff
@@ -0,0 +1,25 @@
+LOGO_URL = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+LOGO_TRADEMARK_URL = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+LOGO_URL_PNG = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+FAVICON_URL = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+
+MFE_CONFIG["LOGO_URL"] = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+MFE_CONFIG["LOGO_TRADEMARK_URL"] = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+MFE_CONFIG["LOGO_WHITE_URL"] = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+MFE_CONFIG["FAVICON_URL"] = "https://d19mbak9hk3cwy.cloudfront.net/wp-content/uploads/2022/10/edly-site-logo-new.svg"
+
+CORS_ORIGIN_ALLOW_ALL = True
+
+FEATURES['ALLOW_PUBLIC_ACCOUNT_CREATION'] = False
+
+LOGIN_REDIRECT_WHITELIST.append("http://apps.local.edly.io")
+CORS_ORIGIN_WHITELIST.append(
+    "http://apps.local.edly.io")
+CSRF_TRUSTED_ORIGINS.append("http://apps.local.edly.io")
+ALLOWED_HOSTS.append("http://apps.local.edly.io")
+
+ENABLE_DYNAMIC_REGISTRATION_FIELDS = "true"
+MFE_CONFIG["ENABLE_DYNAMIC_REGISTRATION_FIELDS"] = "true"
+REGISTRATION_EXTRA_FIELDS["country"] = "required"
+REGISTRATION_EXTRA_FIELDS["terms_of_service"] = "required"
+MFE_CONFIG["TOS_LINK"] = "https://edly.io"
```

Comment:
This works here but there is another patch called `mfe-lms-development-settings` which does the same thing but in terms of making things separate, it is good practice to add MFE_CONFIG changes in there